packages/amplify-graphql-model-transformer/rds-lambda/clients/postgres/PostgresClient.ts

```diff
@@ -1,93 +1,93 @@
 import { Knex } from 'knex';
```

Comment:
As we are not going to test postgres with this release, I recommend not to include these files now. Atleast we should add `TODO`s with proper comments for future work.

Reply:
removed